Formula/dvc.rb

```diff
@@ -23,6 +22,7 @@ class Dvc < Formula
   depends_on "apache-arrow"
   depends_on "libgit2"
   depends_on "libpython-tabulate"
+  depends_on "numpy"
```

Comment:
`numpy` still uses Python 3.9.

Reply:
I see, that is a bumper.

Reply:
`numpy` should support both Python since #97725

Reply:
Ah, whoops. Forgot about that. This should be fine then.